### DISCUSSION TITLE
After deploying of my react project  there are errors of fetching media files

### DISCUSSION THREAD

**Discussion by hovEdgar** [community/community#27963] [Other Feature Feedback, Questions, & Ideas]
I deployed my react project sample on github pages but there is an error of fetching media files, on the localhost everthing works fine!
Can someone help me to fix this issue 

here is the link of website https://hovEdgar.github.io/traveler
and here is link of my repo https://github.com/hovEdgar/traveler

thanks
![Screenshot 2022-08-02 012658](https://user-images.githubusercontent.com/98833505/182250033-35f613e7-81b9-4d08-bb45-08e0c2b8eff0.png)
!

**Comment by airtower-luna** [comment]
The code seems to have an issue with building image links, the scripts on your site request images under `https://hovedgar.github.io/images/`. That this is outside your `https://hovedgar.github.io/traveler/` site, from looking at the workflow logs for deployment `https://hovedgar.github.io/traveler/images/` should work (because images get deployed there).

Most likely you need to make your code build the URLs relative to the current path, instead of just the hostname.

**Reply by hovEdgar** [reply]
could you explain how i can do that please?

**Comment by jfullstackdev** [comment]
if you already updated this, this is what I'm seeing now 

![image](https://user-images.githubusercontent.com/47092464/182289304-86d09fe7-3d9b-4ee3-ab99-9575333b23e3.png)

so I think, it's fixed by now

**Reply by hovEdgar** [reply]
No, i didn't updated yet, its an other error, i dont know why the page loads not from home page, try to go home page and you will see errors
thank you.

**Reply by hovEdgar** [reply]
i fixed all problems but in not standart way, i don't know how git-hub pages works this kind strange, but i forced to do stuped things to make it work, like moving media files from public folder to src 👎
